iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/KubernetesVerifier.java

```diff
@@ -206,6 +217,15 @@ private static String retrieveContent(InputFile inputFile) {
     }
   }
 
+  private static String createFileInBaseDir(String basePath, String content) {
+    try {
+      Files.writeString(BASE_DIR.resolve(basePath).resolve("test.yaml"), content, StandardCharsets.UTF_8);
```

Comment:
Maybe a more unique filename should be used here? I can imagine someone may create a `test.yaml` for some check, and then it will be overwritten. What about: `temp-file-for-verify-k8s-content.yaml` or something similar?